<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>index</class>
 <widget class="QWidget" name="index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{border:one;
}
QWidget{
border:&quot;rgb(122,197,196)&quot;;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>311</width>
     <height>111</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color:rgb(122,196,197);
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>41</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>61</horstretch>
      <verstretch>61</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-radius:14px;
border-width:0px;
border-style:solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>42</y>
      <width>109</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size:23px;
}
</string>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sign">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>78</y>
      <width>106</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>个性签名</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>-5</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
border-image:url(&quot;:/images/myappico.ico&quot;)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>178</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>一</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>318</width>
     <height>441</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>318</width>
     <height>384</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget::tab{
background-color: rgba(203, 203, 203,250);
border-style:solid;
border-bottom-width:2px;
border-radius:0px;
height:32px;
width:106px;
color: rgb(100, 100, 100);
border-color: rgb(255, 255, 255);
border-bottom-color:rgba(163, 163, 163,250);
}
QWidget::tab:selected{
border-bottom-color: rgb(80, 80, 80);
border-bottom-width:2px;
border-radius-:2px;
}
QWidget::pane{
border-style:solid;
background-color: rgba(203, 203, 203,250);
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <attribute name="title">
     <string>消息</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea_3">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>318</width>
          <height>407</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QListWidget" name="listWidget">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>联系人</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>9</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">::tab{
background-color: rgba(203, 203, 203,200);
border-style:solid;
border-width:0px;
border-radius:2px;
height:25px;
width:50px;
color: rgb(100, 100, 100);
border-color: rgb(255, 255, 255);
}
::tab:selected{
border-radius:2px;
}
:pane{
}</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>好友</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QScrollBar:vertical{
background-color:rgba(0,0,0,0);
border-color:rgba(0,0,6,0);
border-style:solid;
border-width:0px;
width:10px;
padding-top:0px;
padding-bottom:0px;
}


QScrollBar::handle:vertical{
border-style:solid;
border-width:2px;
border-radius:4px;
background-color:rgba(145,145,145,255);
border-color:rgba(0,0,0,0);
min-height:44px;}


QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical{background:rgba(0,0,0,0);}


QScrollBar::sub-page:vertical,QScrollBar::sub-page:vertical{background:rgba(0,0,0,0);}</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#scrollAreaWidgetContents{
background-color: rgb(203, 203, 203);
}</string>
            </property>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="geometry">
              <rect>
               <x>9</x>
               <y>9</y>
               <width>299</width>
               <height>361</height>
              </rect>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>群聊</string>
        </attribute>
        <widget class="QScrollArea" name="scrollArea_4">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>291</width>
           <height>371</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>291</width>
            <height>371</height>
           </rect>
          </property>
          <widget class="QToolButton" name="toolButton_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>291</width>
             <height>81</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton{
	background-color:rgb(126,197,197);
}</string>
           </property>
           <property name="text">
            <string>相亲相爱一家人</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>空间</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>318</width>
       <height>415</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>318</width>
        <height>415</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget_2">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget::Item{
background-color: rgb(173, 173, 173);
}

QListWidget::Item:hover{
background-color: rgba(203, 203, 203, 100);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>321</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolButton" name="toolButton">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
background-color:rgb(113,183,193);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background-color:rgb(113,183,193);
}</string>
      </property>
      <property name="placeholderText">
       <string>搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(203, 203, 203,200);
border-top-width:1px;
border-top-color:rgba(163, 163, 163,250);
border-style:solid;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_4">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
border-style:solid;
border-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/images/sangangy.png</normaloff>:/images/sangangy.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_3">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
border-style:solid;
border-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/images/tianjiahaoyou.png</normaloff>:/images/tianjiahaoyou.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>index</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>index</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
